concepts/eventing/performance evaluation/nats-jetstream/results_prod_HA.md

```diff
@@ -0,0 +1,240 @@
+# High Availibiltiy (HA) Evaluation of JetStream (Kyma: Production Profile)
+
+## Table of Contents
+- Test Setup
+- Anti-Affinity
+- Test Scenario 1: Eventing-nats-1 (Stream Leader) pod deleted during test
+- Test Scenario 2: K8s Cluster Node deleted during test where stream leader NATS pod is deployed
+- Test Scenario 3: K8s Cluster Node deleted during test where stream leader NATS pod is NOT deployed
+- Test Scenario 4: Eventing-nats-1 (Stream Leader) pod deleted during test but with Stream Replicas set to 3
+- Conclusion
+
+## Test Setup
+* Testing tool: [K6](https://k6.io/)
+* Kyma CLI version: `2.2.0`
+* Kyma: 
+  * Version: main [[commit](https://github.com/kyma-project/kyma/commit/f8a0c28a43e9eebf192514acc61614300f9909a1)] 
+  * Production Profile
+  * JetStream with File Storage
+* NATS Image: `eu.gcr.io/kyma-project/external/nats:2.8.2-alpine`
+* K8s cluster:
+  * Kubernetes v1.21.10
+  * Gardener cluster [Nodes: 3(min) to 6(max)]
+  * GCP machine type: `n1-standard-4`
+  * Cluster region: `gcp / europe-west1 / europe-west1-c`
+
+* Kyma deploy command:
+  ```
+  kyma deploy --source=main -p production --value global.jetstream.enabled=true --value global.jetstream.storage=file
+  ```
+
+
+## Anti-Affinity
+
+The NATS pods have the AntiAffinity ([here](https://github.com/kyma-project/kyma/blob/6eba6d3b62c15c69645f8577d5c8ca46beeb2c93/resources/eventing/charts/nats/values.yaml#L144)) :
+```
+affinity:
+  podAntiAffinity:
+    preferredDuringSchedulingIgnoredDuringExecution:
+    - podAffinityTerm:
+        labelSelector:
+          matchLabels:
+            nats_cluster: eventing-nats
+        topologyKey: kubernetes.io/hostname
+      weight: 100
+```
+
+> **NOTE:** Note that the Pod anti-afinity policy is `preferredDuringSchedulingIgnoredDuringExecution` as compared to `requiredDuringSchedulingIgnoredDuringExecution` ([see difference here](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#node-affinity)). Therefore, in some cases a single node could have multiple NATS Pods deployed on it.
+
+
+## Test Scenario 1: Eventing-nats-1 (Stream Leader) pod deleted during test
```

Comment:
There is already a lot of information in the heading. Having an introduction sentence to the experiments could help understanding them better :). Something like:
In this experiment we want to delete the NATS steam leader pod using kubectl to figure out something ...

Reply:
Also it would be interesting to know why you chose a particular experiment. Probably you had something in your mind then selection them to draw certain conclusions :)

Reply:
I added a `Test Cases` section before the tests to explain how we reach these specific scenarios.